spec/requests/project_transfer_api_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+describe DDS::V1::ProjectTransfersAPI do
+  include_context 'with authentication'
+  let(:project) { FactoryGirl.create(:project) }
+  let(:other_project) { FactoryGirl.create(:project) }
+  let(:project_transfer) { FactoryGirl.create(:project_transfer) }
+  let(:to_user) { FactoryGirl.create(:user) }
+  let(:other_project_transfer) { FactoryGirl.create(:project_transfer) }
+  let(:project_transfer_stub) { FactoryGirl.create(:project_transfer) }
+  let(:project_transfer_permission) { FactoryGirl.create(:project_permission, :project_admin, user: current_user, project: project) }
+  let(:pending_project_transfer) { FactoryGirl.create(:project_transfer, :pending) }
+
+  let(:resource_class) { ProjectTransfer }
+  let(:resource_serializer) { ProjectTransferSerializer }
+  let!(:resource) { project_transfer }
+  let!(:resource_id) { resource.id }
+  let!(:resource_permission) { project_transfer_permission }
+  let(:resource_stub) { project_transfer_stub }
+
+
+  describe 'Project Transfer Collection' do
+    let(:url) { "/api/v1/projects/#{project_id}/transfers" }
+    let(:project_id) { project.id }
+
+    describe 'POST' do
+      subject { post(url, payload.to_json, headers) }
+      let(:called_action) { 'POST' }
+      let(:payload) {{
+        to_users: [
```

Comment:
We should test that a validation error is thrown when to_users is empty. This may end up throwing a Grape validation error instead of an ActiveRecord validation error. Either is fine, though I'm not sure if the 'a validated resource' shared example will work well with the former.